client/lib/analytics/tracker-buckets.ts

```diff
@@ -65,6 +65,32 @@ export const AdTrackersBuckets: { [ key in AdTracker ]: Bucket | null } = {
 	adroll: null,
 };
 
+const checkGtagInit = (): boolean => 'dataLayer' in window && 'gtag' in window;
+
+export const AdTrackersInitGuards: Partial< { [ key in AdTracker ]: () => boolean } > = {
+	ga: checkGtagInit,
+	gaEnhancedEcommerce: checkGtagInit,
+	floodlight: checkGtagInit,
+	googleAds: checkGtagInit,
+	fullstory: () => 'FS' in window,
+	hotjar: () => 'hj' in window,
+	bing: () => 'uetq' in window,
+	outbrain: () => 'obApi' in window,
+	pinterest: () => 'pintrk' in window,
+	twitter: () => 'twq' in window,
+	facebook: () => 'fbq' in window,
+	linkedin: () => '_linkedin_data_partner_id' in window,
+	criteo: () => 'criteo_q' in window,
+	quora: () => 'qp' in window,
+	adroll: () => 'adRoll' in window,
+};
+
+const isTrackerIntialized = ( tracker: AdTracker ): boolean => {
+	const guardFunction = AdTrackersInitGuards[ tracker ];
+	// If there is no guard function, assume the tracker is initialized
```

Comment:
This is mostly out of curiosity since I'm not familiar with trackers. Why should we assume that the tracker is initialized if there's no guard function?

Reply:
It's more that if we won't define the guard for the certain tracker, the check will be skipped. I'll improve the comment here.